tests/testthat/test-misty.R

```diff
@@ -79,3 +79,27 @@ test_that("run_misty handles tests of failures",{
   expect_true(any(grepl("R2", sig.warnings)))
   unlink("results", recursive = TRUE)
 })
+
+test_that("modeling of intraview is bypassed if only 1 var in intraview", {
+  truncated_expr <- generate_random_tibble(100, 1)
+  pos <- sample_grid_geometry(100, 10, 10)
+  misty.views <- create_initial_view(truncated_expr)
+  sig.warnings <- capture_warnings(misty.views %>% run_misty())
+  expect_true(any(grepl("Negative performance", sig.warnings)))
```

Comment:
This test doesn't check if intraview is bypassed. It only checks for unrelated warning.

Reply:
I was not sure how to check whether the intraview is bypassed when we only have one variable. I just thought that if bypass intra would not work, ranger::ranger would throw an error since there are no covariates (if .norvar = 0 is not added).

Reply:
So could we just check wether run_misty() runs without an error if the input is a tibble with a single column?

Reply:
.novar should be added in this case. yes, simply checking it runs will be better.